### PR TITLE
AC-1100 Catch badly put newlines in LESS files

### PR DESCRIPTION
**Pull request by svera**
Catch newlines in the middle of a style definition and throw an error if any.

Resolves https://github.com/magento/magento-coding-standard/issues/242

### CODE REVIEW

**Review by eliseacornejo — APPROVED** [review]

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the fix @svera ! Please see my comments.

Also, can you please add more information to the phpdoc, particularly it would be great to have a better description for what is happening in the `needValidateSpaces` function

File: Magento2/Sniffs/Less/ColonSpacingSniff.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ public function register()
     public function process(File $phpcsFile, $stackPtr)
     {
         $tokens = $phpcsFile->getTokens();
-
+
```
[the remark this answers is not in the document]
Comment (svera) [reply Magento2/Sniffs/Less/ColonSpacingSniff.php:42]: This are added by PHPStorm, let me check how to remove them

File: Magento2/Sniffs/Less/ColonSpacingSniff.php [hunk 2]
```diff
@@ -56,12 +56,7 @@ public function process(File $phpcsFile, $stackPtr)
      */
     private function needValidateSpaces(File $phpcsFile, $stackPtr, $tokens)
     {
-        $nextSemicolon = $phpcsFile->findNext(T_SEMICOLON, $stackPtr);
-
-        if (false === $nextSemicolon
```
[the remark this answers is not in the document]
Comment (svera) [reply Magento2/Sniffs/Less/ColonSpacingSniff.php:61]: @sivaschenko I think it is more correct to put this in `validateSpaces`, as `needValidateSpaces` is a method that just check if `validateSpaces` should be executed or not

**Review by sivaschenko — APPROVED** [review]

**Comment by sivaschenko** [comment]
@magento import pr to magento-commerce/magento-coding-standard

**Comment by magento-engcom-team** [comment]
@sivaschenko the pull request successfully imported.

**Pull request merged**